llarp/service/endpoint.cpp

```diff
@@ -636,8 +636,7 @@ namespace llarp
     std::set<RouterID>
     Endpoint::GetExitRouters() const
```

Comment:
to be removed.

Reply:
and it was. 
and tom saw it and saw that it was good.